### DISCUSSION TITLE
Are Passthrough presets deprecated?

### DISCUSSION THREAD

**Discussion by ahsan2649** [primefaces/community#2884] [PrimeVue Templates] [answered]
If I understand correctly, previous version implemented Aura and Lara styles using passthrough, and as of 4.1.1 it uses a single style implemented using `@apply`. Will there be more presets using `@apply` or is this the only one (is it Aura or Lara)? Will passthrough presets be implemented further or is it fully upto the user from now on?

**Comment by cagataycivici** [comment, marked as answer]
We do not have any plans to add more presets using @apply since the idea is to offer a single base that you can customize further. The Tailwind version is meant to be customized so one base makes more sense.

PT presets still function, no breaking changes but the way forward we'll maintain the @apply version, because it is easier for us to maintain and share between all Prime libs.
